docs/hyscale-spec-reference.md

```diff
@@ -495,6 +495,20 @@ List of k8s snippets that needs to be patched on the generated manifest files.
 List to define what all services can connect to the service and to which ports
    </td>
   </tr>
+  <tr>
+   <td><a href="#loadBalancer">loadBalancer</a>
+   </td>
+   <td>list
+   </td>
+   <td>
+   </td>
+   <td><em>Optional</em> 
+<p>   
+    <em>Can be overridden</em>
```

Comment:
Since this can be overridden via profile, add this field in profile documentation also